src/api/java/ic2/api/energy/prefab/BasicEnergyTile.java

```diff
@@ -220,7 +220,7 @@ public boolean useEnergy(double amount) {
 	 * @return true if energy was transferred
 	 */
 	public boolean charge(ItemStack stack) {
-		if (stack == null || !Info.isIc2Available() || getWorldObj().isRemote) return false;
+		if (stack == null || stack.isEmpty() || !Info.isIc2Available() || getWorldObj().isRemote) return false;
```

Comment:
Just noticed that's in IC2's api, and probably not used by Mekanism. Shouldn't really have changed it at all tbh